Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -35,7 +35,9 @@ Write-DosMessage -Level "Information" -Message "Using install.config: $installCo
 $installSettingsScope = "identity"
 $installSettings = Get-InstallationSettings $installSettingsScope -installConfigPath $installConfigPath
 
-$idpssConfig = Get-WebConfigPath -service "IdentityProviderSearchService" -discoveryServiceUrl $installSettings.discoveryService -noDiscoveryService $noDiscoveryService -quiet $quiet
+$idpssName = "IdentityProviderSearchService"
```

Comment:
This is assuming IdentityProviderSearchService app name never changes, is that a valid assumption?

Reply:
The name registered with Discovery should be safe to assume it won't change. We can get the app pool details by mapping from the discoveryservice url to IIS config, rather than assuming the app pool name matches the app name....

Reply:
Not sure I follow your statement about app pool names matching